presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/KinesisConfig.java

```diff
@@ -0,0 +1,325 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kinesis;
+
+import io.airlift.configuration.Config;
+import io.airlift.units.Duration;
+
+import javax.validation.constraints.NotNull;
+
+import java.util.concurrent.TimeUnit;
+
+public class KinesisConfig
+{
+    /**
+     * The schema name to use in the connector.
+     */
+    private String defaultSchema = "default";
+
+    /**
+     * Folder holding the JSON description files for Kinesis streams.
+     */
+    private String tableDescriptionLoc = "etc/kinesis/";
```

Comment:
Don't abbreviate variable names

Reply:
If this is supposed to be a local path, consider using `File` instead of `String`

Reply:
Changed so that this can be both S3 and local path